client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
```

Comment:
I ran into a similar question about the usage here. Are we storing the `sandbox` to keep it from being garbage-colelcted during the tests?